modules/ocf_hpc/manifests/controller.pp

```diff
@@ -24,4 +24,15 @@
       File['/etc/slurm-llnl/cgroup.conf']
     ],
   }
+
+  # Set up script to autoadd LDAP group members to SLURM.
+  file { '/usr/local/bin/add_slurm_users':
+    source => 'puppet:///modules/ocf_hpc/add_slurm_users.py',
+    owner  => 'slurm',
```

Comment:
Is this user added by anything in puppet? If so, then an extra dependency should probably be added here so the file and cronjob aren't added before the user actually exists. Anyway, not a big deal on this though.

Reply:
The slurm user is added when the "slurm-wlm" package is installed in the ocf_hpc class, so I think the "require ocf_hpc" line in controller.pp should take care of it.